src/frontend/src/pages/GanttPage/GanttPageWrapper.tsx

```diff
@@ -7,45 +7,46 @@ import LoadingIndicator from '../../components/LoadingIndicator';
 import { useAllProjects } from '../../hooks/projects.hooks';
 import ErrorPage from '../ErrorPage';
 import { Task } from './GanttPackage/types/public-types';
-import { useAllWorkPackages } from '../../hooks/work-packages.hooks';
 import PageTitle from '../../layouts/PageTitle/PageTitle';
 import { Project, WbsElementStatus, WorkPackage } from 'shared';
 import GanttPage from './GanttPage';
 import { projectWbsPipe, wbsPipe } from '../../utils/pipes';
+import GanttPageFilter from './GanttPageFilter';
+import { ChangeEvent, FC, useEffect, useMemo, useState } from 'react';
+import { SelectChangeEvent } from '@mui/material/Select';
+import { useQuery } from '../../hooks/utils.hooks';
+import { useHistory } from 'react-router-dom';
+import { filterGanttProjects, buildGanttSearchParams, GanttFilters } from '../../utils/gantt.utils';
+import { routes } from '../../utils/routes';
+import { useToast } from '../../hooks/toasts.hooks';
 
 /**
  * Documentation for the Gantt package: https://github.com/MaTeMaTuK/gantt-task-react
  */
-const GanttPageWrapper: React.FC = () => {
-  const { isLoading: projectIsLoading, isError: projectIsError, data: projects, error: projErr } = useAllProjects();
-  const { isLoading: wpIsLoading, isError: wpIsError, data: workPackages, error: wpErr } = useAllWorkPackages();
+const GanttPageWrapper: FC = () => {
+  const toast = useToast();
```

Comment:
I'm on mobile so maybe that's why I can't figure it out but what is the toasts being used for?

Reply:
if the date entered is invalid